app/(app)/tools/body-scan/julia-player.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import MediaPlayer from '@/components/MediaPlayer';
+import { ScreenWithImageHeader } from '@/components/ScreenWithImageHeader';
+import { Stack, useFocusEffect, useRouter } from 'expo-router';
+import { YStack } from 'tamagui';
+import { Icon } from '@/components/Icon';
+import { useTranslation } from 'react-i18next';
+import useTranslationKeys from '@/hooks/useTranslationKeys';
+import { useAssetsManagerContext } from '@/contexts/AssetsManagerContextProvider';
+import { useToolManagerContext } from '@/contexts/ToolManagerContextProvider';
+
+export const BodyScanJuliaPlayer = () => {
+  const router = useRouter();
+  const { t } = useTranslation('tools');
+
+  const [mediaURI, setMediaURI] = useState<string | null>(null);
+
+  const { toolsTranslationKeys } = useTranslationKeys();
+  const { mediaMapping } = useAssetsManagerContext();
+
+  useFocusEffect(() => {
+    setMediaURI(mediaMapping['BODY_SCAN.JULIA.soundURI']);
+
+    return () => {
+      setMediaURI(null);
+    };
+  });
+
+  const { finishTool } = useToolManagerContext();
+
+  return (
+    <>
+      <Stack.Screen options={{ headerShown: false }} />
```

Comment:
this can be added in the layout of the current stack, as well as for the other players